File: JavaScript/8 kyu/8 kyu - Convert number to reversed array of digits.js
// https://www.codewars.com/kata/convert-number-to-reversed-array-of-digits/train/javascript
// My solution
function digitize(n) {
    return ("" + n).split("").reverse().map(a => +a)
}

// ...
function digitize(n) {
    return String(n).split('').map(Number).reverse()
}

File: Haskell/8 kyu/8 kyu - Convert number to reversed array of digits.hs
-- https://www.codewars.com/kata/convert-number-to-reversed-array-of-digits/train/haskell
-- My solution
module Codewars.Kata.Convert where
    import Data.Char(digitToInt)
    
    digitize :: Int -> [Int]
    digitize s = map (read . pure :: Char -> Int) $ reverse $ show s

-- ...
module Codewars.Kata.Convert where
    import Data.Char
    
    digitize :: Int -> [Int]
    digitize = reverse . map digitToInt . show

-- ...
module Codewars.Kata.Convert where

    digitize :: Int -> [Int]
    digitize = map (read . return) . reverse . show
